src/components/Comments.jsx

```diff
@@ -0,0 +1,46 @@
+import{ useState, useEffect } from "react";
+import { getArticleComments } from "../api";
+
+const Comments = ({ article_id }) => {
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        setIsLoading(true);
```

Comment:
because the default you have here is true, on a re render loading would be set back to true. You don't need this line 9